src/styles/button.css

```diff
@@ -0,0 +1,19 @@
+.card__button-link {
+  text-align: center;
+  font-size: 14px;
+  line-height: 40px;
+  font-weight: var(--weight-bold);
+  text-decoration: none;
+
+  background-color: var(--blue-accent-color);
+  border: 1px solid;
+  border-radius: 5px;
+
+  color: white;
+}
+
+.card__button-link:hover {
+  background-color: var(--background-color);
+  border-color: var(--blue-accent-color);
+  color: var(--blue-accent-color);
+}
```

Comment:
If a class name has a prefix with the word `card`, thus it should be placed in a file with names `card.css`